elastic/add_elastic_doc.py

```diff
@@ -5,6 +5,7 @@
 es = Elasticsearch([{'host': 'localhost', 'port': 9200, 'scheme': 'http'}])
 
 # Create new index
+#come back and fix, throws error if its the first time run
 es.indices.delete(index='monster_index')
```

Comment:
Could you check if the monster_index index exists in elastic, and if so, delete that index? would that fix the error?